dosemu2.spec.in

```diff
@@ -17,6 +17,7 @@ URL: www.github.com/stsp/dosemu2
 Source0: %{name}-%{version}.tar.gz
 
 BuildRequires: SDL2-devel
+BuildRequires: fontconfig-devel
 BuildRequires: SDL-devel
```

Comment:
Dont we need SDL_ttf-devel here as well?

Reply:
Fedora packaging is always a bit of a guessing game for me! I've added it now.